packages/name-controller/src/NameController.ts

```diff
@@ -40,6 +51,7 @@ export type ProposedNamesEntry = {
 export type NameEntry = {
   name: string | null;
   sourceId: string | null;
+  origin: ORIGIN | null;
```

Comment:
I've been thinking about the default value for our ORIGIN enum, specifically when the caller did not specify an origin while creating a petname. We've got a couple of options:
	1.	Use "UNSPECIFIED" as the default value. This clearly indicates that the origin hasn't been defined by the caller.
	2.	Use "API" as the default. This could imply that the origin is our API by default, which might not be accurate.
My suggestion is to use "UNSPECIFIED" as it seems to be more transparent about the lack of input. However, I'd love to hear your thoughts on this.

Reply:
Done.